### ISSUE TITLE
Uploading attachments to TestRail

### ISSUE THREAD

**Issue by MaksimKravchuk** [gurock/trcli#81]
### What would you like the TestRail CLI to be able to do?

I want to be able to upload attachments like logs, screenshots, etc., to the TestRail test results.

### Why is this feature necessary on the TestRail CLI?

Testrail CLI could be a good tool to gather all tests' results in one place, but when tests fail you need a reason, not only "failed" bage.

### More details

We need to parse a new key from xUnit report. Something like `attachments` : ["path_to_attachment_1.json", "path_to_attachment_2.json"]. 
Then upload the files to test results.

### Interested in implementing it yourself?

Maybe, let's talk!

**Comment by MaksimKravchuk** [comment]
@d-rede Hello! Could you please pay some attention to this issue? I also tried to connect with you via email.

**Comment by d-rede** [comment]
Hi @MaksimKravchuk 

I received your email and I really appreciate that you're trying to contribute to the TestRail CLI.

Before going through with any implementation, could you provide an xUnit report example where we could see the format you are proposing for the attachment paths? And also a code sample of how you're achieving that result on your test code?

Thanks!

**Comment by MaksimKravchuk** [comment]
```xml
<?xml version="1.0" encoding="utf-8"?>
<testsuites>
    <testsuite name="pytest" errors="0" failures="1" skipped="0" tests="1" time="51.913"
               timestamp="2022-06-30T17:43:33.610906" hostname="Maks-Kravcuk--MacBook-Pro.local">
        <testcase classname="test_main_flow_demo" name="test_example_is_working" time="51.826">
            <properties>
                <property name="attachments"
                          value="['tests/logs/playwright_demo.test_main_flow_demo/test_example_is_working/trace-test_example_is_working.zip', 'tests/logs/playwright_demo.test_main_flow_demo/test_example_is_working/test_example_is_working.log']"/>
            </properties>
            <failure message="AssertionError: This test should fail&#10;assert False">page = &lt;Page
                Here is some assertion error message
                test_main_flow_demo.py:43: AssertionError
            </failure>
        </testcase>
    </testsuite>
</testsuites>
```
To implement this with Pytest we use https://docs.pytest.org/en/7.1.x/reference/reference.html#record-property
```python
@pytest.fixture(autouse=True)
def add_attachments_to_report(request, record_property, log_path):
    """Add attachments to the report."""
    record_property(
        "attachments",
        [
            f"{log_path}/trace-{request.node.name}.zip",
            f"{log_path}/{request.node.name}.log",
        ],
    )
```
I've already made some changes, which allows me to parse attachment's paths. But I face trouble: when I run trying to debug trcli with PyCharm Environment is passed between commands. So I need some help with IDE configurations.

**Comment by MaksimKravchuk** [comment]
@d-rede Hi! I open PR. Could you please review it?

**Comment by d-rede** [comment]
Hi @MaksimKravchuk 

I'll take a look at this within the next 2/3 days. 

Thanks for contributing and also for your patience :)

**Comment by d-rede** [comment]
Hey everyone,

This feature is now available on [TestRail CLI v1.3.0](https://github.com/gurock/trcli/releases/tag/v1.3.0).
Please check the README for information on how it works and let me know if anything weird comes up! :)

**Comment by Testinator-X** [comment]
@d-rede there is a small typo in the readme example code

```
     <properties>
         <property name="testrail_attachment" value="/path_to_/logs.log"/>
         <property name="testrail_attachment" value="/path_to_/screenshot.jpg"/>
     </properties>>
```

(Double closing > of the properties tag)
